Fix `make run-cleanup`

Currently, `make run-cleanup` does not seem to work. It fails with:
```
/app/node_modules/ts-node/src/index.ts:500
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
../tsconfig.json:16:9 - error TS5023: Unknown compiler option 'noImplicitOverride'.

16         "noImplicitOverride": true,
           ~~~~~~~~~~~~~~~~~~~~
../tsconfig.json:18:9 - error TS5023: Unknown compiler option 'noPropertyAccessFromIndexSignature'.

18         "noPropertyAccessFromIndexSignature": true,
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

    at createTSError (/app/node_modules/ts-node/src/index.ts:500:12)
    at reportTSError (/app/node_modules/ts-node/src/index.ts:504:19)
    at create (/app/node_modules/ts-node/src/index.ts:515:36)
    at Object.register (/app/node_modules/ts-node/src/index.ts:398:19)
    at main (/app/node_modules/ts-node/src/bin.ts:162:19)
    at Object.<anonymous> (/app/node_modules/ts-node/src/bin.ts:512:3)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
```
This should be fixed.

Also, (I think at least the last time I tried it) this does only work if the container is not currently running (at least it build the container again even if it is running, which is unnecessary). We should rename it to `make run-cleanup-standalone` (or similar) and add a new `make run-cleanup` which `exec`s into the running client container and executes the cleanup directly.

--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,5 @@
+docker-run=docker run -ti -v `pwd`/client/src:/app/src -p 127.0.0.1:9001:9001/tcp openslides-client-dev
+
 build-dev:
 	docker build -t openslides-client-dev -f Dockerfile.dev .
 
@@ -5,13 +7,16 @@ build-prod:
 	docker build -t openslides-client -f Dockerfile .
 
 run-dev: | build-dev
-	docker run -ti -v `pwd`/client:/app -p 127.0.0.1:9001:9001/tcp openslides-client-dev
+	$(docker-run)
 
 run-dev-interactive: | build-dev
-	docker run -ti -v `pwd`/client:/app -p 127.0.0.1:9001:9001/tcp openslides-client-dev sh
+	$(docker-run) sh
 
-run-cleanup: | build-dev
-	docker run -t -v `pwd`/client:/app openslides-client-dev npm run cleanup
+run-cleanup-standalone: | build-dev
+	$(docker-run) npm run cleanup
+
+run-cleanup:
+	docker exec -it $$(docker ps -a -q  --filter ancestor=openslides-client-dev) npm run cleanup
 
 run-tests: | build-dev
 	docker run -t openslides-client-dev npm run lint

--- a/README.md
+++ b/README.md
@@ -137,6 +137,8 @@ See https://editorconfig.org/ for details.
 
 Code can be cleaned and aligned automatically using `npm run cleanup`.
 This will take care of code alignment, import sorting and quotation marks.
+To execute this inside the docker container, you can either use `make run-cleanup` while the client
+container is already running or `make run-cleanup-standalone` if it's not.
 
 ## Translation
 

--- a/client/src/app/site/pages/meetings/pages/motions/pages/workflows/components/workflow-detail/workflow-detail.component.html
+++ b/client/src/app/site/pages/meetings/pages/motions/pages/workflows/components/workflow-detail/workflow-detail.component.html
@@ -195,7 +195,7 @@
             *ngFor="let restriction of restrictions"
             (click)="onSetRestriction(restriction.key, state)"
         >
-            <mat-icon *ngIf="state.restrictions.includes(restriction.key)">check</mat-icon>
+            <mat-icon *ngIf="state.restrictions?.includes(restriction.key)">check</mat-icon>
             <span>{{ restriction.label | translate }}</span>
         </button>
     </ng-template>

--- a/client/src/app/site/pages/meetings/pages/motions/pages/workflows/components/workflow-detail/workflow-detail.component.ts
+++ b/client/src/app/site/pages/meetings/pages/motions/pages/workflows/components/workflow-detail/workflow-detail.component.ts
@@ -292,7 +292,7 @@ export class WorkflowDetailComponent extends BaseMeetingComponent {
      * @param state the state to change
      */
     public onSetRestriction(restriction: Restriction, state: ViewMotionState): void {
-        const restrictions = state.restrictions.map(r => r);
+        const restrictions = [...(state.restrictions ?? [])];
         const restrictionIndex = restrictions.findIndex(r => r === restriction);
 
         if (restrictionIndex < 0) {

